features/step_definitions/brexit_check_steps.rb

```diff
@@ -0,0 +1,47 @@
+When "I start the checker" do
+  visit_path "/get-ready-brexit-check"
+  click_link "Start now"
+end
+
+When "I answer the nationality question" do
+  choose('British', visible: false)
```

Comment:
Minor, but I don't think we need the `visible: false` modifiers for these non-invisible elements.

Reply:
Need as the radio buttons are hidden to support custom styling.